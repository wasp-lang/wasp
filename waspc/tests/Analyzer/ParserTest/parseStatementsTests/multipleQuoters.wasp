test JSON {=json
  "key": "value"
json=}
test JSON2 {=json [1, 2, 3] json=}
