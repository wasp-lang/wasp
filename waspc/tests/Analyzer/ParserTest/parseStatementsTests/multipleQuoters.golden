(AST
  (Decl@1:1-3:6 type=test name=JSON
    (Quoter@1:11-3:6 tag=json
      {=
        "key": "value"
      =}
    )
  )
  (Decl@4:1-34 type=test name=JSON2
    (Quoter@4:12-34 tag=json
      {= [1, 2, 3] =}
    )
  )
)
